cellarium/ml/core/module.py

```diff
@@ -114,17 +139,45 @@ def configure_model(self) -> None:
                 )
             )
         else:
-            transforms = None
+            transforms = tuple()
 
-        self.pipeline = CellariumPipeline(transforms)
         if model is None:
```

Comment:
Why check for `None` and not `~isinstance(model, CellariumModel)`?

Reply:
Yeah, it makes more sense to change it to `not isinstance(model, CellariumModel)`.